mirrord/config/src/lib.rs

```diff
@@ -19,12 +19,26 @@ use std::path::Path;
 use config::{ConfigError, MirrordConfig};
 use mirrord_config_derive::MirrordConfig;
 use schemars::JsonSchema;
+use tracing::warn;
 
 use crate::{
     agent::AgentConfig, config::source::MirrordConfigSource, feature::FeatureConfig,
     target::TargetConfig, util::VecOrSingle,
 };
 
+const PAUSE_WITHOUT_STEAL_WARNING: &str =
+    "--pause specified without --steal: Incoming requests to the application will
+                                           not be handled. The target container running the deployed application is paused,
+                                           and responses from the local application are dropped.
+
+                                           Attention: if network based liveness/readiness probes are defined for the
+                                           target, they will fail under this configuration.
+
+                                           To have the local application handle incoming requests you can run again with
+                                           `--steal`. To have the deployed application handle requests, run again without
+                                           specifying `--pause`.
+    ";
```

Comment:
I think I did something weird there with the indentation. Can we fix it now?
Maybe like this:
```suggestion
    "--pause specified without --steal: Incoming requests to the application will
    not be handled. The target container running the deployed application is paused,
    and responses from the local application are dropped.

    Attention: if network based liveness/readiness probes are defined for the
    target, they will fail under this configuration.

    To have the local application handle incoming requests you can run again with
    `--steal`. To have the deployed application handle requests, run again without
    specifying `--pause`.
    ";
```